app/components/Vault/index.js

```diff
@@ -1268,84 +1276,83 @@ const Vault = (props) => {
       );
     }
   }
-  return (
-    ((vault.balanceOf > 0 &&
-      migrationVaultsFrom.includes(vault.address.toLowerCase())) ||
-      !migrationVaultsFrom.includes(vault.address.toLowerCase())) && (
-      <React.Fragment>
-        <Card
-          className={`vault ${amplifyVault ? 'amplify-vault' : ''} ${
-            active ? 'active' : ''
-          } ${vault.isYVBoost ? 'pickle-vault' : ''}`}
-          id={`vault-${accordionKey}`}
+
+  return (!vaultsToHide.includes(vault.address.toLowerCase()) ||
```

Comment:
I think we should filter in a component one level higher as returning `undefined` from a react component is not the best possible practice.

Reply:
It's ok to return null https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

We could go one level higher but we'd need to move all the balance logic one element higher too which would take time.